src/reanimated2/NativeReanimated.js

```diff
@@ -1,10 +1,15 @@
-import { TurboModuleRegistry } from 'react-native';
+import { TurboModuleRegistry, Platform } from 'react-native';
+import reanimatedJS from './js-reanimated';
 
 const InnerNativeModule =
-  global.__reanimatedModuleProxy ||
-  TurboModuleRegistry.getEnforcing('NativeReanimated');
+  Platform.OS === 'web'
+    ? false
+    : global.__reanimatedModuleProxy ||
```

Comment:
Could you add parentheses here? It's not clear if it's evaluated like `(A ? B : C) || D` or `A ? B : (C || D)`

Reply:
eslint reverts this anyway, could disable it of course, but this looks fine to me(not ambiguous)